### DISCUSSION TITLE
[10:49:25] Error while executing SQL query on database 'test': no such table: test

### DISCUSSION THREAD

**Discussion by bruna1418** [pawelsalawa/sqlitestudio#3914] [Q&A]
Hello,

Could you please help me with this error on SQLLiteStudio?

I'm trying to execute this simple command:
SELECT * FROM test; 
When I execute this error show up:
[10:49:25] Error while executing SQL query on database 'test': no such table: test
<img width="959" alt="Error while executing SQL query on database test o such table test" src="https://user-images.githubusercontent.com/77446765/104599544-b348c900-5656-11eb-9352-098fb0f3b156.png">

**Comment by pawelsalawa** [comment]
You do not have `test` table. The `test` in your case is a name of database. You can select from `item`, or `sale` or `customer`.

**Reply by elderbr** [reply]
You can remove the database from the SQLITE manager list and then add it back to make it work again.
I noticed that accessing the database from another application causes this error.
